File: cakezone-backend/src/main/java/com/lakshithafdo/cakezone_backend/Service/OrderService.java
package com.lakshithafdo.cakezone_backend.Service;

import com.lakshithafdo.cakezone_backend.Entity.Order;

import java.util.List;

public interface OrderService {
    List<Order> getAllOrders();

    Order createOrder(Order order);

    Order getOrdersById(long id);

    Order updateOrder(Order order, long id);
}

File: Cakezone/src/Services/order.service.ts
import axios from 'axios';
import { Order } from '../Models/Order';

const ORDER_API_BASE_URL = "http://localhost:8080/api/v1/orders";


    export const getAllOrders = () =>{
        return axios.get(ORDER_API_BASE_URL);
    }

    export const createOrder = (order:Order) => {
        return axios.post(ORDER_API_BASE_URL, order);
    }

    export const getOrdersById = (id:number)=>{
        return axios.get(ORDER_API_BASE_URL + '/' + id);
    }

    // updateOrder(newOrder, id){
    //     return axios.put(EMPLOYEE_API_BASE_URL + '/' + employeeId, employee);
    // }

    // deleteEmployee(employeeId){
    //     return axios.delete(EMPLOYEE_API_BASE_URL + '/' + employeeId);
    // }

